YouTubeChannels.md

```diff
@@ -78,6 +78,9 @@
 * [Scott Hanselman](https://www.youtube.com/channel/UCL-fHOdarou-CR2XUmK48Og): Scott has over 30 years of experience in the industry, 15 of which have been spent at Microsoft. He's written a number of technical books and spoken in person to over one million developers worldwide! He's also been podcasting and blogging everything tech for over 20 years at [www.hanselman.com](https://www.hanselman.com/).
 Working in Open Source on .NET and the Azure Cloud for Microsoft out of his home office in Portland, Oregon, he gives us a glimpse of his day-to-day life as a developer on his YouTube channel: a great source of tips & tricks to make your life as a programmer easier. His [PowerShell configuration video](https://www.youtube.com/watch?v=VT2L1SXFq9U) is a must!
 
+* [Net Ninja](https://www.youtube.com/@NetNinja): Net Ninja is a seasoned developer focused on sharing his knowledge on full-stack development.
+He helps in developing the basics so you can move on and start taking on projects on your own. He teaches Javascript, PHP, Flutter, React, Vue and many more.
```

Comment:
@ASANIYAN Please could you merge line 81 and 82 together so that they aren't 2 separate lines? After this, maybe one of the maintainers will merge your PR 🙂